### ISSUE TITLE
Asset modal?

### ISSUE THREAD

**Issue by cole007** [pixelandtonic/AssetSubfolderAccess#3]
Does this add-on only work within the Assets 'tab' of the control panel or should it work within asset modals (when creating/editing an entry).

Can only seem to get it to work in former instance.

**Comment by maxwellhibbert** [comment]
Yeah, it would be awesome if this worked in the modal. Kinda useless otherwise..
